### PR TITLE
Display collateralization ratio in showMarketDetails

### PR DESCRIPTION
**Pull request by mrice32**
Note: there are other places where we might want to show the collateralization ratio. The primary purpose of this PR was to get the price feed infrastructure in a place where it can be used from the CLI tool.

To do this, this PR provides a way of generating "default" reference price feeds for identifiers. This means, for most use cases, a config is unnecessary.

Related to #1529

### CODE REVIEW

File: core/scripts/cli/sponsor/showMarketDetails.js [hunk 1]
```diff
@@ -28,6 +33,36 @@ const showMarketDetails = async (web3, artifacts, emp) => {
...
+        }
+
+        if (!priceFeed) {
+          return "Unknown";
+        }
+
+        await priceFeed.update();
+        const collateralizationRatio = await computeCollateralizationRatio(
+          web3,
+          priceFeed.getCurrentPrice(),
```
Comment (nicholaspai) [inline core/scripts/cli/sponsor/showMarketDetails.js:58]: If the pricefeed is constructed properly, should we expect that `getCurrentPrice()` should occasionally fail and return `null`? If so, then we should handle that either before calling `computeCollateralizationRatio` or in that method.

Comment (mrice32) [reply core/scripts/cli/sponsor/showMarketDetails.js:58]: Good point. Will handle that case.

Comment (mrice32) [reply core/scripts/cli/sponsor/showMarketDetails.js]: Fixed.

File: common/EmpUtils.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/**
```
Comment (chrismaree) [inline common/EmpUtils.js:1]: this util is super useful. There are a few places in the bots that could re-use this.

File: financial-templates-lib/price-feed/CreatePriceFeed.js [hunk 3]
```diff
@@ -122,6 +123,64 @@ function isMissingField(config, requiredFields, logger) {
...
+ * Create a reference price feed for the EMP. Note: this is the price feed that the token is tracking.
+ * @param {Object} winston logger.
+ * @param {Object} web3 instance.
+ * @param {Object} networker object that the price feed may use to make REST calls.
+ * @param {Function} function to get the current time.
+ * @param {String} string representing the address of the EMP contract.
+ * @param {Object} optional config to override the defaults for this reference feed.
+ * @return {Object} an instance of PriceFeedInterface that can be used to get the reference price.
+ */
+async function createReferencePriceFeedForEmp(logger, web3, networker, getTime, empAddress, config) {
```
Comment (chrismaree) [inline financial-templates-lib/price-feed/CreatePriceFeed.js:136]: Would we want to refactor the bots to use this if no price feed information is provided? is a useful default.

Comment (mrice32) [reply financial-templates-lib/price-feed/CreatePriceFeed.js:136]: Yep, that's the idea. I want us to try to reduce the amount of config parameters we provide across the board. The price feeds are a big chunk of that.

**Pull request merged**